pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -525,6 +525,13 @@ func (f *framework) runFilterPlugin(ctx context.Context, pl FilterPlugin, state
 // RunPostFilterPlugins runs the set of configured PostFilter plugins until the first
 // Success or Error is met, otherwise continues to execute all plugins.
 func (f *framework) RunPostFilterPlugins(ctx context.Context, state *CycleState, pod *v1.Pod, filteredNodeStatusMap NodeToStatusMap) (*PostFilterResult, *Status) {
+	preemptionStartTime := time.Now()
+	defer func() {
+		metrics.PreemptionAttempts.Inc()
+		metrics.SchedulingAlgorithmPreemptionEvaluationDuration.Observe(metrics.SinceInSeconds(preemptionStartTime))
+		metrics.DeprecatedSchedulingDuration.WithLabelValues(metrics.PreemptionEvaluation).Observe(metrics.SinceInSeconds(preemptionStartTime))
+	}()
+
```

Comment:
this should be inside the preemption plugin. We need a separate PostFilter extension point metric like the others.

Reply:
Do you mean the metric defined in `runPostFilterPlugin()`?

Reply:
No, I am talking about `FrameworkExtensionPointDuration`, see RunScorePlugins for example.

```PreemptionEvaluation``` should only be measuring the default preemption logic, and so we should move it inside the plugin. We can later deprecate this metric since this overhead will be covered by the metric set in `runPostFilterPlugin`.

Reply:
Done. PTAL.

cc/ @alculquicondor

Reply:
Sounds good. PreemptionEvaluation latency is already covered by the extension point latency metric, and so PreemptionEvaluation can be deprecated later. But we don't have a PreemptionAttempts equivalent counter, should we have one for extension point attempts?

Reply:
+1 to deprecating Preemption latency in the future.